pkg/solana/config/config.go

```diff
@@ -17,7 +17,8 @@ var defaultConfigSet = Chain{
 	OCR2CacheTTL:        config.MustNewDuration(time.Minute),            // stale cache deadline
 	TxTimeout:           config.MustNewDuration(time.Minute),            // timeout for send tx method in client
 	TxRetryTimeout:      config.MustNewDuration(10 * time.Second),       // duration for tx rebroadcasting to RPC node
-	TxConfirmTimeout:    config.MustNewDuration(30 * time.Second),       // duration before discarding tx as unconfirmed
+	TxConfirmTimeout:    config.MustNewDuration(30 * time.Second),       // duration before discarding tx as unconfirmed. Set to 0 to disable discarding tx.
```

Comment:
- What would be the use case of disabling discarding a tx?
- Could we have a case where we stack confirmed transactions that don't go through and we are unable to drop them? 
- Could this cause a memory leak and should we worry about it?

Reply:
- I made this disable-able for the new fee bumping strategy actually. Dropping transactions actually only really works for Data Feeds where there's a follow up update anyways but for CCIP and other products we would want to get transactions through to finalization
- What do you mean by stacking confirmed transactions? I should mention that we probably only expect to disable this if we're using the new fee bumping strategy where we wait for expiration before sending a new tx. So we would have another mechanism to drop transactions before sending a new one so txs wouldn't stack.
- Glad you're thinking about that. I think if we can reliably pick up on transaction expiration/re-orgs and handle those cases then we wouldn't have to worry about memory leaks

Reply:
Thanks for the clarification!